rust/src/eval/function/mod.rs

```diff
@@ -62,6 +66,32 @@ fn log(args: Vec<serde_json::Value>) -> Result<serde_json::Value> {
     }
 }
 
+fn sum(args: Vec<serde_json::Value>) -> Result<serde_json::Value> {
+    if let Some(val) = args.get(0) {
+        match val {
+            serde_json::Value::Array(arr) => {
+                let mut sum: f64 = 0.0;
+                for item in arr {
+                    match item {
+                        serde_json::Value::Number(num) => {
+                            sum += num.as_f64().unwrap();
```

Comment:
Need to use `match` here. If one of the array elements is NaN, it will match `serde_json::Value::Number` but `as_f64()` will return `None`, and `unwrap()` will panic.